package.json

```diff
@@ -1,68 +1,106 @@
+		"start": "webpack --env.development --env.watch",
+		"zip": "gulp zip",
+		"prettier": "prettier '**/*.{js,jsx,ts,tsx}'",
+		"format-check": "npm run prettier -- --check",
+		"format": "npm run prettier -- --write",
+		"prettier-json": "prettier '**/*.json'",
+		"format-check-json": "npm run prettier-json -- --check",
+		"format-json": "npm run prettier-json -- --write",
+		"eslint": "eslint '**/*.{js,jsx,ts,tsx}'",
+		"lint-check": "npm run eslint -- --quiet",
+		"lint-all": "npm run eslint -- --fix",
+		"lint": "npm run lint-all -- --quiet",
+		"lint-check-typed": "npm run lint-check -- -c ./.eslintrc.typed.js --no-eslintrc",
+		"lint-all-typed": "npm run lint-all -- -c ./.eslintrc.typed.js --no-eslintrc",
+		"lint-typed": "npm run lint -- -c ./.eslintrc.typed.js --no-eslintrc",
+		"tsc": "tsc --noEmit",
+		"check": "run-s format-check-json tsc lint-check-typed",
+		"fix": "run-s format-json tsc lint-typed"
+	},
+	"dependencies": {
+		"@material-ui/core": "^4.9.1",
+		"@material-ui/icons": "^4.9.1",
+		"@material-ui/lab": "^4.0.0-alpha.41",
+		"history": "^4.10.1",
+		"moment": "^2.24.0",
+		"prop-types": "^15.7.2",
+		"react": "^16.12.0",
+		"react-dom": "^16.12.0",
+		"react-router-dom": "^5.1.2",
+		"rollbar": "^2.14.4",
+		"typeface-roboto": "^0.0.75",
+		"webextension-polyfill": "^0.6.0"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.8.4",
+		"@babel/plugin-proposal-class-properties": "^7.5.5",
+		"@babel/plugin-transform-runtime": "^7.8.3",
+		"@babel/preset-env": "^7.8.4",
+		"@babel/preset-flow": "^7.8.3",
+		"@babel/preset-react": "^7.8.3",
+		"@babel/preset-typescript": "^7.6.0",
+		"@babel/runtime": "^7.8.4",
+		"@octokit/rest": "^16.43.1",
+		"@svgr/webpack": "^5.1.0",
+		"@types/gulp": "^4.0.6",
+		"@types/history": "^4.7.5",
+		"@types/material-ui": "^0.21.7",
+		"@types/node": "^14.0.13",
+		"@types/react": "^16.9.23",
+		"@types/react-dom": "^16.9.5",
+		"@types/react-router-dom": "5.1.3",
+		"@typescript-eslint/parser": "^3.3.0",
+		"@typescript-eslint/eslint-plugin": "^3.3.0",
+		"babel-eslint": "^10.0.3",
+		"babel-loader": "^8.0.6",
+		"babel-preset-minify": "^0.5.1",
+		"clean-webpack-plugin": "^3.0.0",
+		"css-loader": "^3.4.2",
+		"eslint": "^7.3.0",
+		"eslint-config-prettier": "^6.11.0",
+		"eslint-plugin-prettier": "^3.1.4",
+		"eslint-plugin-react": "^7.20.0",
+		"file-loader": "^5.0.2",
+		"fs-extra": "^8.1.0",
+		"gulp": "^4.0.2",
+		"gulp-zip": "^5.0.1",
+		"husky": "^4.2.5",
+		"lint-staged": "^10.2.11",
+		"npm-run-all": "^4.1.5",
+		"prettier": "^2.0.5",
+		"progress-bar-webpack-plugin": "^2.1.0",
+		"react-svg-loader": "^3.0.3",
+		"string-replace-loader": "^2.2.0",
+		"style-loader": "^1.1.3",
+		"typescript": "^3.6.3",
+		"web-ext-types": "^3.2.1",
+		"webpack": "^4.41.5",
+		"webpack-cli": "^3.3.10"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"lint-staged": {
+		"*.json": "prettier --write",
+		"*.{js,jsx,ts,tsx}": [
```

Comment:
prettier should run on these files as well?

Reply:
It does. `eslint-plugin-prettier` shows Prettier errors as ESLint errors and automatically fixes them when run with `--fix`. So when you run ESLint, it's automatically running Prettier as well. That's why I separated .json above, because ESLint can't run on .json. Here's how we configurate the plugin: https://github.com/trakt-tools/universal-trakt-sync/blob/prettier-eslint/.eslintrc.js#L13